### PR TITLE
LPS-138104 Extract getDisplayName to Locale file To reuse on several components

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by Hanlf** [comment]
The above failures are not related to this PR.

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
The root cause of the above failures was reverted here: https://github.com/liferay/liferay-portal/compare/78e7911baadcc5baafca8a05b0a7db73c8c1c6f0...da5431eca174cb11ca689cd49ae7f4bd0616751a, re-run it.

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
DatabasePartitioningUpgrade#CanUpgradePartitionedDatabase741 failed at LPS-138315.
PortalSmokeUpgrade#ViewPortalSmokeArchive621021 failed at LPS-136151
Fragments Poshi tests failure should be flaky.

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
The failures are not related to this PR, let me know if you want me to send it to BChan directly @AliciaGarciaGarcia

**Comment by adolfopa** [comment]
@Hanlf Not necessary for this. This is a refactor/cleanup we want to send when the release is done.

File: portal-kernel/src/com/liferay/portal/kernel/util/LocaleUtil.java [hunk 1]
```diff
@@ -128,6 +128,10 @@ public static LocaleUtil getInstance() {
 		return _localeUtil._getISOLanguages(locale);
 	}
 
+	public static String getLocaleDisplayName(Locale curLocale, Locale locale) {
```
Comment (adolfopa) [inline portal-kernel/src/com/liferay/portal/kernel/util/LocaleUtil.java:131]: Why not rename "curLocale" to "displayLocale" as in the private method?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: portal-kernel/src/com/liferay/portal/kernel/util/LocaleUtil.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply portal-kernel/src/com/liferay/portal/kernel/util/LocaleUtil.java:131]: perfect, thanks for pointing at it

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by Hanlf** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:stop

**Pull request closed without merging**